tests/unittests/GraphOptzTest.cpp

```diff
@@ -84,6 +84,14 @@ static Function *optimizeFunction(Function *F) {
   return G;
 }
 
+template <typename NodeT = Node>
+static const NodeT *findFunctionNodeByName(const Function *f,
+                                           const char *name) {
```

Comment:
Why `const char*`? Why not `const std::string&` or even better yet, ` const llvm::StringRef`?

Reply:
fixed